SHELLY_MJS.md

```diff
@@ -2,6 +2,11 @@ activation-switch.js: Activation switch
 ===
 Replicate activation_switch profile from Gen1 devices.
 
+BLE in Scripting - Ruuvi example
+===
+Example how to use a 3rd party BLE sensor, read data and turn on switch based on the data. (Requires firmware version 
```

Comment:
we can be more specific that this is ruuvi sensor

Reply:
<img width="885" alt="Screenshot 2022-11-08 at 17 37 13" src="https://user-images.githubusercontent.com/139877/200608637-bcb95981-e0b9-40d3-b771-a7b38052922f.png">

Its in the title :?